bmorph/evaluation/StatCor_Vis.py

```diff
@@ -0,0 +1,566 @@
+            hyears = hyears + 1
+        i = i + 1
+        
+    diff_maxflow_sites_df = pd.DataFrame(columns = observed.columns, index = pd.Series(range(0,hyears)))
+    diff_maxflow_current_year = pd.DataFrame(columns = observed.columns)
+    
+    for i in range(0,hyears):
+        #establish end of hydraulic year
+        hyear_end = hyear_start + pd.Timedelta(364.25, unit = 'd')
+        
+        #need to truncate datetimes since time indicies do not align
+        O = observed.loc[hyear_start:hyear_end]
+        P = predicted.loc[hyear_start:hyear_end]
+        O.index = O.index.floor('d')
+        P.index = P.index.floor('d')
+        
+        diff_maxflow_current_year = P.max().to_frame().T - O.max().to_frame().T
+        
+        #puts the computations for the hydraulic year into our bigger dataframe
+        diff_maxflow_sites_df.iloc[i] = diff_maxflow_current_year.iloc[0].T
+        
+        #set up next hydraulic year
+        hyear_start = hyear_start + pd.Timedelta(365.25,unit = 'd')
+
+    
+    return diff_maxflow_sites_df
+
+def pbias_plotter(observed: pd.DataFrame, names: list, colors: list, *models: pd.DataFrame):
+    """
+    Percent Bias Box Plotter
+        plots box plots of numerous models grouped
+        by site
+    ----
+    observed: pd.Dataframe
+        a dataframe containing observations
+    names: list
+        a list of the model names
+    colors: list
+        a list of colors to be plotted
+    *models: any number of pd.DataFrame objects
+        to be evaluated
+    """
+    num_models = len(models)
+    sites = observed.columns
+    pbias_models = list()
+    
+    #runs each model through pbias_sites
+    for model in models:
+        pbias_models.append(pbias_sites(observed,model))  
+        
+    fig = plt.figure()
+    ax = plt.axes(xlabel = 'Sites', ylabel = 'Percent Bias')
+    position = 1
+    for site in sites:
+        df = pd.DataFrame(index = pbias_models[0].index)
+        
+        #fill out the dataframe for a single site with each model's percent bias
+        i = 0
+        for model in pbias_models:
+            entry = f"{site}:{names[i]}"
+            df[entry] = model[site]
+            i = i+1
+    
+        boxplots = plt.boxplot(df.T, positions = np.arange(position,position+num_models),patch_artist=True)
+         
+        for patch, color in zip(boxplots['boxes'], colors):
+            patch.set_facecolor(color)
+        
+        position = position+num_models+1
+        #follow plot style: https://stackoverflow.com/questions/16592222/matplotlib-group-boxplots
+   
+    tick_locaction = list()
+    start_tick = int(np.ceil(len(models)/2))
+    tick_spacing = len(models)+1
+    for j in range(0,len(sites)):
+        tick_location.append(start_tick+j*tick_spacing)
+    
+    ax.set(xticks = tick_location, xticklabels = sites)
+    plt.xticks(rotation = 45)
+        
+    ax.legend(handles=custom_legend(names,colors),loc='upper left')
```

Comment:
Maybe add a `return fig, ax` here to make it easier for the user to continue to modify the plot.

Reply:
`ax.set(xticks = tick_location, xticklabels = sites)
 plt.xticks(rotation = 45)
        
 ax.legend(handles=custom_legend(names,colors),loc='upper left')
 return fig, ax`

Like so?

Reply:
Yep, exactly